src/components/Content/TitledCategoriesTable.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import PageModel from 'endpoints/models/PageModel'
+import CategoriesTable from './CategoriesTable'
+import LOCATIONS_ENDPOINT from 'endpoints/location'
+import withFetcher from 'endpoints/withFetcher'
+
+import style from './TitledCategoriesTable.css'
+
+class TitledCategoriesTable extends React.Component {
+  static propTypes = {
+    parentPage: PropTypes.instanceOf(PageModel).isRequired,
+    pages: PropTypes.arrayOf(PropTypes.shape({
+      page: PropTypes.instanceOf(PageModel).isRequired,
+      url: PropTypes.string.isRequired
+    })).isRequired
+  }
+
+  getTitle () {
+    return this.props.locations.find((location) => location.code === this.props.parentPage.title).name
+  }
+
+  render () {
+    return (
+      <div>
+        <div className={style.title}>{this.getTitle()}</div>
+        <CategoriesTable pages={this.props.pages}/>
+      </div>
+    )
+  }
+}
+
+export default withFetcher(LOCATIONS_ENDPOINT, true, true)(TitledCategoriesTable)
```

Comment:
It seems, we currently cannot have two instances of the same fetcher in two different spots at the same time (here: LocationsFetcher in Breadcrumbs and in TitledCategoriesTable).
A possible workaround for now would be to use the LocationFetcher in the LocationPage-Route in the FetchingContentWrapper and pass the title down to the Breadcrumbs and the Content.
But I would actually prefer, that the Fetchers care about that logic, so we can easily put Fetchers where we want to.
I think we need to discuss this with @maxammann (preferably on Wed)?

Reply:
It seems, the actual problem occurs, e.g. when transitioning from Augsburg (Home), to Augsburg-> Welcome: At Welcome the breadcrumbs should appear, but dont. The problem is that the TitledCategoriesTable's fetcher invalidates the locations endpoint data and then the breadcrumbs are hidden by the its locationsFetcher (since the payload-state has changed by invalidation).

Reply:
going to test this

Reply:
Created issue: https://integreat.atlassian.net/browse/WEBAPP-74

Reply:
works well with the solution of maxamman, WEBAPP-74